Updating CAD and BOM to reflect slip clutch additions (#3)
htem/GridTapeStage: CHANGES.md

> Dec 2023 - PR #3

Reviewer: ```suggestion
# Dec 2023 - [Updating CAD and BOM to reflect slip clutch changes](https://github.com/htem/GridTapeStage/pull/3)
```

> Hardware

Reviewer: ```suggestion
## Hardware
```

> - Parts

Reviewer: ```suggestion
### Parts
```

> 	- Updated the reel_drive_clamp and shaft_adapter CAD to their slip clutch varients
> 	- Added slip clutch and slip clutch pulley CAD 
> 	- Updated overall CAD assembly to include slip clutch and slip clutch adjacent CAD

Reviewer: ```suggestion

- Updated the reel_drive_clamp and shaft_adapter CAD to their slip clutch varients
- Added slip clutch and slip clutch pulley CAD 
- Updated overall CAD assembly to include slip clutch and slip clutch adjacent CAD
- Added missing counterbore to back of `reel_hub`

```

Is `reel_hub` the right part?

Author: Nope, it's reel_adapter.

```suggestion
- Updated the reel_drive_clamp and shaft_adapter CAD to their slip clutch varients
- Added slip clutch and slip clutch pulley CAD 
- Updated overall CAD assembly to include slip clutch and slip clutch adjacent CAD
- Added missing counterbore to back of `reel_adapter'
```

> - Drawings

Reviewer: ```suggestion
### Drawings
```

	- Added drawings of updated CAD

> Docs

Reviewer: ```suggestion
## Docs
```

> - Updated BOM:

Reviewer: ```suggestion
### BOM
```

> 	- Updated CAD components
> 	- Changed the material of the takeup_rest from aluminum to stainless steel
> 	- Included 4-pin connectors for the load cell amplifier and the LED, and a 7pos housing receptacle for the LED buckpuck

Reviewer: ```suggestion

- Updated CAD components
- Changed the material of the takeup_rest from aluminum to stainless steel
- Included 4-pin connectors for the load cell amplifier and the LED, and a 7pos housing receptacle for the LED buckpuck

```
Would you expand `Updated CAD components` above?

Author: Yeah! 
```suggestion
- Changed the reel drive plate and slip clutch shaft CAD file reference to reel_drive_clamp_slipclutch and shaft_adatper_slipclutch
- Removed components for old slipclutch
- Added current slipclutch
- Changed the material of the takeup_rest from aluminum to stainless steel
- Included 4-pin connectors for the load cell amplifier and the LED, and a 7pos housing receptacle for the LED buckpuck
```

> - Updated Gridtape Assembly Instructions

Reviewer: ```suggestion
### Assembly Instructions

details about the assembly instructions changes...
```
